packages/shared/src/utils/config/config.js

```diff
@@ -1,31 +1,30 @@
 // const Cookies = require('js-cookie');
-const isBot = require('Utils/PlatformSwitcher/platform-switcher').isBot;
+import { isBot } from '../platform/platform';
 
 /*
  * Configuration values needed in js codes
  *
  * NOTE:
  * Please use the following command to avoid accidentally committing personal changes
- * git update-index --assume-unchanged src/javascript/config.js
+ * git update-index --assume-unchanged packages/shared/src/utils/config.js
  *
  */
-const domain_app_ids = {
+export const domain_app_ids = {
     // these domains as supported "production domains"
     'deriv.app': 16929,
 };
+const binary_desktop_app_id = 14473;
 
-const getCurrentProductionDomain = () =>
+export const getCurrentProductionDomain = () =>
     !/^staging\./.test(window.location.hostname) &&
     Object.keys(domain_app_ids).find(domain => new RegExp(`.${domain}$`, 'i').test(window.location.hostname));
 
-const isProduction = () => {
+export const isProduction = () => {
     const all_domains = Object.keys(domain_app_ids).map(domain => `www\\.${domain.replace('.', '\\.')}`);
```

Comment:
Could you sneak in a fix here for isProduction too? 🙏currently it's returning false.

```suggestion
    const all_domains = Object.keys(domain_app_ids).map(domain => `(www\\.)?${domain.replace('.', '\\.')}`);
```

Reply:
Done.